core/auth/interfaces.go

```diff
@@ -237,6 +237,8 @@ type APIs interface {
 	CreateAdminAccount(authenticationType string, appID string, orgID string, identifier string, profile model.Profile, privacy model.Privacy, username string, permissions []string,
 		roleIDs []string, groupIDs []string, scopes []string, creatorPermissions []string, clientVersion *string, l *logs.Log) (*model.Account, map[string]interface{}, error)
 
+	//CreateAdminAccounts creates an accounts for a new admin user
```

Comment:
The descprition is wrong. it should be clear that it creates accounts. For example //CreateAccounts create accounts in the system